app/helpers/template_helpers.py

```diff
@@ -57,12 +58,19 @@ def render_template(template, **kwargs):
         get_locale().language, theme or "census"
     )
     page_header_context.update({"title": cookie_session.get("survey_title")})
-    google_tag_mananger_context = get_google_tag_mananger_context()
+    google_tag_manager_context = get_google_tag_manager_context()
     cdn_url = f'{current_app.config["CDN_URL"]}{current_app.config["CDN_ASSETS_PATH"]}'
     contact_us_url = get_contact_us_url(theme, get_locale().language)
+    include_csrf_token = True if cookie_session.get(USER_IK) is not None else False
+    account_service_url = (
+        cookie_session.get("account_service_url")
+        if cookie_session
+        else "https://census.gov.uk/en/start"
```

Comment:
This could use the get base url method?.

Reply:
It should prob also default to the base URL even if you have a cookie session but no account service URL given it is not a required claim.